0a.md

```diff
@@ -0,0 +1,80 @@
+NIP-0a
+======
+
+Audio Events
+---------------
+
+`draft` `optional`
+
+This NIP defines several new event kinds representing different types of audio track. These events
+are _parameterized replaceable_ as defined in [NIP-01](./01.md) and deletable per [NIP-09](09.md).
+
+Unlike a `kind 1` event with audio attached, audio events are intended to be surfaced in an audio-specific context
+rather than in a general micro-blogging context.
+
+The following kinds are defined by this NIP:
+
+- `31337` represents a music track
+- `31338` represents a podcast episode
+
+The following tags are required:
+
+- `d` is a unique identifier randomly generated by the client.
+- `media` is the url of the media. This SHOULD NOT be read, but is required for backwards compatibility reasons.
+- `imeta` is as described in [NIP 92](./92.md). Multiple `imeta` tags MAY be included to provide for different
+  use cases (e.g. streaming vs download).
+- `title` is the title of the audio track
+- `subject` is the title of the audio track (deprecated, but required for compatibility)
+
+The following tags are optional:
+
+- `i` is an external GUID in the format defined [here](https://github.com/MerryOscar/nips/commit/280eb498e0ac56b8f9356c1b7a88cc8b31579801).
+- `c` is the track's value as defined by a given category.
+- `website` is an external url to a website related to the track.
+- `duration` is the duration of the audio track, in seconds.
+- `published_at` is a timestamp representing the track's original publish date.
+
+Expected categories may include (but are not limited to):
+
+- `genre`
+- `subgenre`
+- `artist`
+- `record_label`
+- `producer`
+- `composer`
+- `album`
+- `track_number`
+- `tracks`
+- `bpm`
+- `year`
+
+Note that artists, producers, SHOULD be referred to using both a plaintext `c` tag AND a conventional `p` tag when pubkey is available. When doing so, the `p` tag's petname MUST be the same as the `c` tag's value. In this way `c` tags can be used to indicate genre, as well as artist, producer, etc.
```

Comment:
What are the consequences of a `p` tag's petname not matching a `c` tag's value?

If a pubkey updates their petname, must these audio tracks be replaced with an updated version to reflect the new petname to avoid a mismatch of `p` and `c` tags?

Reply:
Petnames are arbitrary, and aren't assigned by the pubkey owner. If they don't match, then clients would have to treat the `c` tag as an arbitrary string and the `p` tag as a normal mention.